configuration/uart.md

```diff
@@ -1,46 +1,61 @@
+| Model | mini UART | PL011 |
+|-------|-----------|-------|
+| Raspberry Pi Zero | secondary | primary |
+| Raspberry Pi Zero W | primary (disabled) | secondary (Bluetooth) |
+| Raspberry Pi 1 | secondary | primary |
+| Raspberry Pi 2 | secondary | primary |
+| Raspberry Pi 3 | primary (disabled) | secondary (Bluetooth) |
+| Raspberry Pi 4 | primary (disabled) | secondary (Bluetooth) |
 
-The baud rate of the mini UART is linked to the core frequency of the VPU on the VC4 GPU. This means that, as the VPU frequency governor varies the core frequency, the baud rate of the mini UART also changes. This makes the mini UART of limited use in the default state. By default, if the mini UART is selected for use as the primary UART, it will be disabled. To enable it, add `enable_uart=1` to config.txt. This will also fix the core frequency to 250MHz (unless `force_turbo` is set, when it will be fixed to the VPU turbo frequency). When the mini UART is not the primary UART, for example you are using it to connect to the Bluetooth controller, you must add `core_freq=250` to config.txt, otherwise the mini UART will not work.
+Linux devices on Raspbian:
 
-The default value of the `enable_uart` flag depends on the actual roles of the UARTs, so that if ttyAMA0 is assigned to the Bluetooth module, `enable_uart` defaults to 0. If the mini UART is assigned to the Bluetooth module, then `enable_uart` defaults to 1. Note that if the UARTs are reassigned using a Device Tree Overlay (see below), `enable_uart` defaults will still obey this rule.
+| Linux device | Description |
+|--------------|-------------|
+|`/dev/ttyS0`  |mini UART    |
+|`/dev/ttyAMA0`|PL011        |
+|`/dev/serial0` |primary UART (Linux console) |
+|`/dev/serial1` |secondary UART |
 
-## Disabling Linux's use of console UART
+## Mini UART and CPU core frequency
 
-In a default install of Raspbian, the primary UART (serial0) is assigned to the Linux console. Using the serial port for other purposes requires this default behaviour to be changed. On startup, `systemd` checks the Linux kernel command line for any console entries, and will use the console defined therein. To stop this behaviour, the serial console setting needs to be removed from command line.
+The baud rate of the mini UART is linked to the core frequency of the VPU on the GPU. This means that, as the VPU frequency governor varies the core frequency, the baud rate of the mini UART also changes. This makes the mini UART of limited use in the default state. By default, if the mini UART is selected for use as the primary UART, it will be disabled. To enable it, add `enable_uart=1` to config.txt. This will also fix the core frequency to 250MHz (unless `force_turbo` is set, when it will be fixed to the VPU turbo frequency). When the mini UART is not the primary UART, for example you are using it to connect to the Bluetooth controller, you must add `core_freq=250` to config.txt, otherwise the mini UART will not work.
```

Comment:
"selected for use" implies that this is an action performed by the user, whereas it's actually a function of which model of Pi is being used? Perhaps "if the mini UART is being used for the primary UART" would be better? :man_shrugging: